content/blog/creating-sigils.md

```diff
@@ -0,0 +1,139 @@
+
+We felt this was too complicated and had an unknown performance profile because we predicted that a tiling algorithm would need to be used. Plus, they proposed a deterministic color scheme and we felt strongly that color is personal — in the end we thought it better to create something that had the potential for user customization. 
+
+![](https://media.urbit.org/site/posts/essays/sigils4.png)
+
+Around this stage, we were thinking a lot about how architectural plans aren’t unduly constrained by the fact that they’re only one color. We had a feeling we could make the grid smaller and assign details to the tiles individually. 
+
+So, we scaled it down to a 2 x 2 grid and stuck with it. It’s simple to implement, visually unique, consistent, and the color scheme is black and white so users can customize their sigil. Not a bad start. Now all we had to do was manually draw 512 unique but consistent individual ‘phonemes’.
+
+![](https://media.urbit.org/site/posts/essays/sigils5.png)
+
+As it turns out, drawing 512 consistent, abstract elements by hand isn’t easy. We had to strike a balance between authorship and automation — we wanted each tile to look well-made, but not too authored. To do so, we turned to Figma’s web API which helped us design in SVG but iterate programmatically. 
+
+## Final Process
+
+![](https://media.urbit.org/site/posts/essays/sigils6.png)
+
+We took a look at our hand drawn mockups and drew each individual visual building block of the tiles separately. Then we loaded the SVG elements into a script that programmatically combined them to create trillions of layered symbols. This way, we were able to easily select candidate phoneme symbols that worked visually. 
+
+We eventually added a rule system to find higher quality phoneme symbols — but this combination of programmatic generation and hand selection was working pretty well.
+
+![](https://media.urbit.org/site/posts/essays/sigils7.png)
+
+The final step, using the tool we built, was to hand select individual phoneme symbols and create a complete index. 
+
+![](https://media.urbit.org/site/posts/essays/sigils8.jpg)
+
+This allowed us to start looking at possible combinations and selecting phoneme symbols based on their interplay with others. 
+
+Once we had selected the final set, we went about compiling the sigil-js library that’s now publicly available. This turned out to be pretty technically complex — but the end result is a single js library that you can use to embed an SVG in almost any context on a web page. It’s called sigil-js, and you can find it [here](https://github.com/urbit/sigil-js). You can also see it in action and browse the available sigils [here](https://urbit.live/explore).
+
+![](https://media.urbit.org/site/posts/essays/sigils9.png)
+
+We think the sigils are pretty successful. They look pretty nice in our native Urbit interfaces, people have made them into shirts and posters and cards, even laser-cut them to make stamps. 
```

Comment:
Might be good to link or show these but that's a "would be nice" thing

Reply:
Agree — @galenwp are there examples of any higher-quality examples floating around out there?

Reply:
We should probably be collecting them — but I don't have any. I think it's fine without an example.

Reply:
Sounds good to me!

Reply:
Can add this as a general task in the marcom "communities" ongoing todos

Reply:
Ongoing task added here: https://app.asana.com/0/1131086629952617/1160460977063161